web/src/routes/CityNotCooperatingPage.tsx

```diff
@@ -45,7 +44,6 @@ const ListItem = styled.div`
 `
 
 const StepNumber = styled.div`
-  float: left;
```

Comment:
This element is already aligned to the left because the parent is a flexbox with direction row.